src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -61,77 +62,123 @@ DoorLockServer & DoorLockServer::Instance()
  */
 void DoorLockServer::InitServer(chip::EndpointId endpointId)
 {
-    emberAfDoorLockClusterPrintln("Door Lock cluster initialized at %d", endpointId);
+    emberAfDoorLockClusterPrintln("Door Lock cluster initialized at endpoint #%" PRIu16, endpointId);
 }
 
 bool DoorLockServer::SetLockState(chip::EndpointId endpointId, DlLockState newLockState)
 {
     auto lockState = static_cast<uint8_t>(newLockState);
```

Comment:
Pre-existing by the way, but using `to_underlying` here instead of explicit cast would be much better for type-safety.  Similar in other places here.

Reply:
Fixed